### ISSUE TITLE
Flexbox layout

### ISSUE THREAD

**Issue by domoritz** [uwdb/website#9]
Please use flexbox for the logos on the front page. Also, include from another file so that we can use markdown.

**Comment by domoritz** [comment]
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

**Comment by BrandonHaynes** [comment]
I'll be closely mirroring the homepage layout on the project page, so if I can reuse the included list (and CSS) that would be fantastic.

**Comment by domoritz** [comment]
I was talking about the logo list, not the projects list. So you can probably reuse the CSS but not the list itself. I think by "layout" you meant just the general way to do it, but just to make sure that there is no confusion. 

I think flexbox is really useful for the logos on the front page, the projects list, and the photos on the peoples page (cc @jingjingwang). 

![screen shot 2015-03-10 at 11 06 27](https://cloud.githubusercontent.com/assets/589034/6581652/a9e5b412-c715-11e4-9e31-5c8a67b63594.png)

**Comment by domoritz** [comment]
The css will probably be something like

``` css
.flexcontainer {
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: row;
   flex-wrap: wrap;
   flex-flow: row wrap;
   align-content: flex-end;
}
```

**Comment by BrandonHaynes** [comment]
Oops -- missed that. In that case I'll sub-include the current projects and flexbox them, and Shumo/Brenden can reuse that on the homepage.

**Comment by domoritz** [comment]
Great. Whatever works best.

**Comment by stechu** [comment]
@domoritz  how can I set CSS for a specific markdown section?

**Comment by domoritz** [comment]
Maybe with

``` md
<p class='specialParagraph' markdown='1'>
**Another paragraph** which allows *Markdown* within it.
</p>
```

Or just use plain html (like above) but use an include to get markdown inside of it.

**Comment by stechu** [comment]
This seems break the markdown image compilation.
Neither inline or include works.
![screenshot 2015-03-10 11 57 28](https://cloud.githubusercontent.com/assets/720571/6582772/cd4a09b0-c71c-11e4-8d8b-9f4e0e6e9639.png)

I think I am just fine to stick to inline html but use flexbox style.

**Comment by stechu** [comment]
It does align the images better though.
![screenshot 2015-03-10 12 01 52](https://cloud.githubusercontent.com/assets/720571/6582844/44faa03c-c71d-11e4-92f2-32cae925919f.png)

**Comment by stechu** [comment]
closed by #9d688e1abee53959606b20016589ff27a3aac48d.
